tst/regression/test_suites/restart/restart.py

```diff
@@ -67,11 +75,11 @@ def Analyse(self, parameters):
 
         success = True
 
-        def compare_files(name):
+        def compare_files(name, base="silver"):
             delta = compare(
                 [
                     "gold.out0.%s.rhdf" % name,
-                    "silver.out0.%s.rhdf" % name,
+                    "{}.out0.{}.rhdf".format(base, name),
```

Comment:
Why not go fully modern? ;)
`f"{base}.out0.{name}.rhdf",`

Reply:
oh, man, I must be getting old. I thought `"".format()` was the modern way of doing it...

Reply:
I'm also still not in the habit of doing fstrings....